web-marketplace/src/components/organisms/Members/Members.tsx

```diff
@@ -0,0 +1,253 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useLingui } from '@lingui/react';
+
+import ContainedButton from 'web-components/src/components/buttons/ContainedButton';
+import DropdownIcon from 'web-components/src/components/icons/DropdownIcon';
+import EditIcon from 'web-components/src/components/icons/EditIcon';
+import EmailIcon from 'web-components/src/components/icons/EmailIcon';
+import SmallArrowIcon from 'web-components/src/components/icons/SmallArrowIcon';
+import { Title } from 'web-components/src/components/typography';
+
+import { ActionsDropdown } from '../Collaborators/ActionsDropdown';
+import { SEE_HELP_DOCS, YOU } from '../Collaborators/Collaborators.constants';
+import {
+  EDIT_PROFILE,
+  INVITE_MEMBERS,
+  NAME,
+  ORGANIZATION_MEMBERS,
+  ORGANIZATION_MEMBERS_DESCRIPTION,
+  ROLE,
+  VISIBILITY_ON_PROFILE,
+} from './Members.constants';
+import { mockMembers } from './Members.mock';
+import { Member, MemberRole } from './Members.types';
+import { MemberRoleDropdown } from './MembersRoleDropdown';
+import { VisibilitySwitch } from './VisibilitySwitch';
+
+export const Members = ({
+  initialMembers = mockMembers,
+}: {
+  initialMembers?: Member[];
+}) => {
+  const { _ } = useLingui();
+  const navigate = useNavigate();
+  const [members, setMembers] = useState<Member[]>(initialMembers);
+  const [sortDir, setSortDir] = useState<'asc' | 'desc'>('asc');
+  const currentUserRole: MemberRole =
+    members.find(m => m.isCurrentUser)?.role ?? 'viewer';
+  const canAdmin = currentUserRole === 'admin';
+  const isOnlyAdmin = members.filter(m => m.role === 'admin').length <= 1;
+
+  const toggleSort = () => {
+    const dir = sortDir === 'asc' ? 'desc' : 'asc';
+    setSortDir(dir);
+    setMembers(prev =>
+      [...prev].sort((a, b) =>
+        dir === 'asc'
+          ? a.name.localeCompare(b.name)
+          : b.name.localeCompare(a.name),
+      ),
+    );
+  };
+
+  const updateRole = (id: string, role: MemberRole) =>
+    setMembers(prev => prev.map(m => (m.id === id ? { ...m, role } : m)));
+
+  const updateVisibility = (id: string, visible: boolean) =>
+    setMembers(prev => prev.map(m => (m.id === id ? { ...m, visible } : m)));
+
+  const handleRemove = (id: string) =>
+    setMembers(prev => prev.filter(m => m.id !== id));
+
+  const handleInvite = () => {
+    /* open invite modal */
+  };
+
+  return (
+    <div
+      className="w-full px-10 py-30 md:p-40 bg-bc-neutral-0 rounded-lg"
+      style={{ border: '1px solid #D2D5D9' }}
+    >
+      {/* Header row */}
+      <div className="flex justify-between items-center mb-10">
```

Comment:
I was assuming we would use the same underlying component for both the members and collaborators tables
there's a lot of repeated code in Members and Collaborators.tsx that could be refactored